charts/newrelic-prometheus/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes are documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Should we wait for https://github.com/newrelic/newrelic-prometheus-configurator/pull/89 to be merged?

Reply:
I would like to have a first version released since the repo is already public, I would prefer not to block the first release until we agree on the toolkit